tasks/post_to_catalog/post_to_catalog.py

```diff
@@ -107,21 +107,12 @@ def create_catalog_records(
                     }
                 ],
             )
-            LOGGER.debug("Creating provider_collection_xref record.")
-            connection.execute(
-                create_provider_collection_xref_sql(),
-                [
-                    {
-                        "provider_id": provider["providerId"],
-                        "collection_id": collection["collectionId"]
-                    }
-                ]
-            )
             LOGGER.debug("Creating granule record '{cumulusGranuleId}'", cumulusGranuleId=granule["cumulusGranuleId"])
             results = connection.execute(
                 create_granule_sql(),
                 [
                     {
+                        "provider_id": provider["providerId"],
```

Comment:
This is the correct way to get the providerId.

Reply:
gotcha